undefined_syms.txt

```diff
@@ -499,6 +499,31 @@ D_060135EC = 0x060135EC;
 D_06013CD8 = 0x06013CD8;
 D_060140BC = 0x060140BC;
 
+// z_boss_sst
+D_06017C40 = 0x06017C40;
+D_0600E7B8 = 0x0600E7B8;
+D_0600B6FC = 0x0600B6FC;
+D_06004DE0 = 0x06004DE0;
+D_0600A350 = 0x0600A350;
+D_060002E8 = 0x060002E8;
+D_06005860 = 0x06005860;
+D_0600CC6C = 0x0600CC6C;
+D_06013D80 = 0x06013D80;
+D_0600C288 = 0x0600C288;
+D_0600DC2C = 0x0600DC2C;
+D_06017EE0 = 0x06017EE0;
+D_06017F80 = 0x06017F80;
+D_040184B0 = 0x040184B0;
+D_0600B0D8 = 0x0600B0D8;
+D_0600ACD4 = 0x0600ACD4;
+D_0600D458 = 0x0600D458;
+D_0600C9BC = 0x0600C9BC;
+D_0600C5B0 = 0x0600C5B0;
+
+// z_bg_sst_floor
+D_060194F8 = 0x060194F8;
+D_06019210 = 0x06019210;
+
```

Comment:
imo it doesnt matter if it stays or goes. you can if you want, but it doesnt really help anything, and can create conflicts if multiple people touch this area

Reply:
Imo we should remove them if they aren't necessary. They will have to be removed at some point, and in this case it won't create conflicts since they aren't in the repo yet.

Reply:
my thought process was moreso that undefined syms as a whole will become irrelevant anyway. but fair enough

Reply:
If you're referring to the file itself, it will probably remain relevant for a long time, because it's used to define many OS symbols among other things. Eventually we can probably move these definitions somewhere else, or find a better solution, but that probably won't happen any time soon. Even the sm64 project still has its `undefined_syms.txt` for the same reason.